lib/screens/create_region_screens/add_region_background_image/components/upload_picture_box.dart

```diff
@@ -28,21 +31,25 @@ class UploadPictureBox extends StatelessWidget {
         color: AppColors.grey,
         child: Ink(
           height: 200,
-          child: Stack(
-            alignment: Alignment.center,
-            children: [
-              if (backgroundImage != null) Image.asset(backgroundImage!),
-              Row(
-                mainAxisAlignment: MainAxisAlignment.center,
-                children: [
-                  const Icon(Icons.add),
-                  const SizedBox(
-                    width: 6,
+          child: Container(
+            width: width,
+            child: backgroundImage != null
+                ? Image.file(backgroundImage!, fit: BoxFit.cover)
+                : Stack(
+                    alignment: Alignment.center,
+                    children: [
+                      Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          const Icon(Icons.add),
+                          const SizedBox(
+                            width: 6,
+                          ),
```

Comment:
Remember  SizedBox is a one line instruction

Reply:
Fix lol